src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,145 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public MyLinkedList() {
+        this.head = null;
+        this.tail = null;
+        this.size = 0;
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (head == null) {
+            Node<T> newNode = new Node<>(value, null, null);
+            head = newNode;
+            tail = newNode;
```

Comment:
this row repeats in else statement as well, you can put before size++, not to write it twice